### PR TITLE
Hadeel - Database week3

### PR DESCRIPTION
**Pull request by Hadeel-Mur**

### CODE REVIEW

File: Week3/homework/assignment/1-SQLnormalization.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+1- What columns violate 1NF?
+none of the columns in the provided table violate the First Normal Form (1NF).
```
Comment (ayatalat) [inline Week3/homework/assignment/1-SQLnormalization.md:2]: wrong answer 
 `dinner_date` has multiple types
  `food_code` , `food_description`  multiple values not atomic

File: Week3/homework/assignment/1-SQLnormalization.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+2- What entities do you recognize that could be extracted?
```
Comment (ayatalat) [inline Week3/homework/assignment/1-SQLnormalization.md:4]: also we can extract 
    member, food, venue, dinner

File: Week3/homework/assignment/1-SQLnormalization.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+venue_description
+number_address
+
+3- Name all the tables and columns that would make a 3NF compliant solution.
+table1 (members) member_id, member_name, member_address.
+table2 (dinners) dinner_id, dinner_date, venue_code, food_code.
```
Comment (ayatalat) [inline Week3/homework/assignment/1-SQLnormalization.md:10]: why do you add venue_code here ?

**Review by ayatalat — APPROVED** [review]

File: Week3/homework/assignment/mongo-practice.js [hunk 4]
```diff
@@ -0,0 +1,277 @@
+// Insert the documents into the 'movies' collection
+
+const { MongoClient} = require('mongodb');
+const uri = "mongodb+srv://murshedhadeel:<2tQWHrrvGDZdIbYY>@cluster0.asbcayp.mongodb.net/?retryWrites=true&w=majority";
```
Comment (ayatalat) [inline Week3/homework/assignment/mongo-practice.js:4]: this should be added from env file and read it from it 
you can use a npm package called require("dotenv") checkout here how you can use it https://www.npmjs.com/package/dotenv

**Review by ayatalat — CHANGES_REQUESTED** [review]

File: Week3/homework/assignment/1-SQLnormalization.md (hunk 3 above)
Comment (Hadeel-Mur) [reply Week3/homework/assignment/1-SQLnormalization.md:10]: avoiding duplicating venue information (such as venue_description and number_address) for each dinner event. Instead,i reference the venue table using a foreign key (venue_code),

**Pull request closed without merging**